### DISCUSSION TITLE
Email function not working with new format for update change feature

### DISCUSSION THREAD

**Discussion by rtillinois** [code4recovery/12-step-meeting-list#1085] [Q&A]
A user just informed me by direct email that when they attempted to send an email through the 12 Step Meeting list plugin they got an error message: Could not instantiate mail function. I tried it and got the same error. I check the Settings and the contact email is the same.  I deleted and added the email contact address, but the result is the same

![image](https://user-images.githubusercontent.com/81121628/222716389-fe4bed54-4463-42e1-b409-7297ea3bd846.png)

**Comment by joshreisner** [comment]
hi, that error is coming from wordpress, not the plugin, so i believe this may be an error on your site

try opening a private window and trying the login > forgot password function. does it work?

if not, i believe you may need to re-setup email on your site

**Reply by joshreisner** [reply]
for what it's worth, hosting companies often worry about wordpress websites getting compromised and sending lots of spam emails, so they do change security policies around this from time to time.

one option is to use a plugin like this to send emails: https://wordpress.org/plugins/wp-mail-smtp/

**Comment by anchovie91471** [comment]
@rtillinois,

FWIW. I've found using the mail() function causes a lot of the messages from my sites to land in the spam folder (including password resets, registration emails, form submission receipts, etc). I never use it anymore.

Using an SMTP plugin ensures better deliverability and isn't very difficult to setup. 

All you need is a valid email box to send from (preferrably the same as your Wordpress Administration Email Address), and the settings for the SMTP server you normally use to send email from that address. 

Most hosting companies provide all this for you since you need it to set up Outlook, Apple Mail or whatever mail program you normally use. The WP Mail SMTP works, I use it all the time. Here's what you need to set it up:

1. The SMTP server name (smtp.somehostingcompany.com)
2. The encryption use on the server (TLS most of the time, but there are a couple other options)
3. SMTP Port (587 most of the time)
4. The email address you'll use to send
5. The password for that email address

You'll be glad you did it!
